deploy/olm-catalog/1.0.2/mercury-operator.csv.yaml

```diff
@@ -131,6 +131,7 @@ spec:
     - name: GitHub Repository
       url: https://github.com/open-accelerators/mercury
   version: 1.0.2
```

Comment:
I think it's better to rename all `1.0.2` to `v1.0.2` that affects the bundle and catalog images too

Reply:
I looked at a lot of other operators the version tag in all of them is without the v prefix, I guess that's the way it should be